spec/shortcuts_spec.rb

```diff
@@ -50,6 +50,32 @@
     expect(m).to eq identity3
   end
 
+  it "hilbert() creates an hilbert matrix" do
+    m = NMatrix.hilbert(3)
+    hilbert3 = NMatrix.new([3, 3], [1.0, 0.5, 0.3333333333333333, 0.5, 0.3333333333333333, 0.25, 0.3333333333333333, 0.25, 0.2])
+    expect(m).to eq hilbert3
+
+    m = NMatrix.hilbert(8)
+    hilbert8 = [ 0.2, 0.16666666666666666, 0.14285714285714285, 0.125,  0.1111111111111111]
+    expect(m[4, 0]).to eq(hilbert8[0])
+    expect(m[4, 1]).to eq(hilbert8[1])
+    expect(m[4, 2]).to eq(hilbert8[2])
+    expect(m[4, 3]).to eq(hilbert8[3])
+  end
+
+  it "inv_hilbert() creates an inverse hilbert matrix" do
+    m = NMatrix.inv_hilbert(6)
+    inv_hilbert6 = [3360.0,  -88200.0,   564480.0, -1411200.0]
+    expect(m[2, 0]).to eq(inv_hilbert6[0])
+    expect(m[2, 1]).to eq(inv_hilbert6[1])
+    expect(m[2, 2]).to eq(inv_hilbert6[2])
+    expect(m[2, 3]).to eq(inv_hilbert6[3])
+
+    m = NMatrix.inv_hilbert(3)
+    inv_hilbert3 = NMatrix.new([3, 3], [  9.0,  -36.0,   30.0, -36.0,  192.0, -180.0, 30.0, -180.0,  180.0] )
+    expect(m).to eq inv_hilbert3
+  end
```

Comment:
It would be easier and more dependable to use `be_within` when comparing floating point numbers.